src/crystal/system/win32/signal.cr

```diff
@@ -5,6 +5,10 @@ module Crystal::System::Signal
     raise NotImplementedError.new("Crystal::System::Signal.trap")
   end
 
+  def self.trap_handler?(signal)
+    nil
```

Comment:
I think this and the wasi one should also raise `NotImplementedError`.

Reply:
updated